src/sorting.cpp

```diff
@@ -133,3 +133,45 @@ std::vector<int> counting_sort(std::vector<int>& vec, int k) {
 
   return result;
 }
+
```

Comment:
I dont know what is best. But should we not have descriptions??
It realy depends in the purpose of the repo. If it is for learning the algorithm then a small description would be nice, but i think that the complexity of the function is nice to have.
But you if it is for learning the coding of algorithms then i don't think the description is to important.